lib/services/debug-service-base.ts

```diff
@@ -28,9 +28,9 @@ export abstract class DebugServiceBase extends EventEmitter implements IDeviceDe
 				return true;
 			}
 		};
-	}
+  }
 
-	protected getChromeDebugUrl(debugOptions: IDebugOptions, port: number): string {
+	protected getChromeDebugUrl(debugOptions: IDebugOptions, port: number, legacy?: boolean): string {
```

Comment:
I think adding `Legacy` here and passing it through the engine adds a lot more code which isn't needed as this url should be removed in the future version.   I would just fix it here to return the proper url with `devtools://`